pkg/orchestration/docker.go

```diff
@@ -195,8 +194,16 @@ func runNode(
 		}
 	}
 
+	if config.Type != conf.Identity {
+		privateKey, err := NormalizedPrivateKey(host, config.PrivateKey)
+		if err != nil {
+			return logger.Error(err)
+		}
+		config.PrivateKey = privateKey
+	}
+	override := config.ToOverrideEnv(host, nconf)
 	// generate the override.env file locally
-	// WARNING: NOT THREAD SAFE
+	// WARNING: not thread safe due to constant filename
```

Comment:
thread safe in the way that multiple audius-ctl commands could be generating the same file?